mix.exs

```diff
@@ -33,7 +33,16 @@ defmodule Membrane.MP4.Plugin.MixProject do
       main: "readme",
       extras: ["README.md"],
       source_ref: "v#{@version}",
-      nest_modules_by_prefix: [Membrane.MP4]
+      nest_modules_by_prefix: [
+        Membrane.MP4,
+        Membrane.MP4.Muxer,
+        Membrane.MP4.Payloader
+      ],
+      groups_for_modules: [
+        Muxers: ~r/Membrane.MP4.Muxer/,
+        Payloaders: ~r/Membrane.MP4.Payloader/,
+        Boxes: ~r/Membrane.MP4.*Box$/
```

Comment:
I've missed that in the original suggestion, but dot here actually means "any character", so the last line matches e.g. `Membrane.MP4Box`. Let's change it to something that actually expresses our intentions :wink: 
```suggestion
        Muxers: ~r/Membrane\.MP4\.Muxer/,
        Payloaders: ~r/Membrane\.MP4\.Payloader/,
        Boxes: ~r/Membrane\.MP4\..+Box$/
```